dependencies.yaml

```diff
@@ -493,7 +493,7 @@ dependencies:
           - requests
           - nccl>=2.9.9
           - ucx-proc=*=gpu
-          - &ucx_py ucx-py==0.39.*,>=0.0.0a0
+          - &ucx_py_unsuffixed ucx-py==0.39.*,>=0.0.0a0
```

Comment:
Following from my other review on the cuDF PR: I don't care if we use `_unsuffixed` or not. `ucx_py` is fine, just so long as it doesn't say `_conda` if it's used in non-conda lists.

Reply:
I'd like to keep the cases like this, I think it's a useful convention that makes the intent slightly clearer.